.circleci/config.yml

```diff
@@ -378,7 +429,8 @@ jobs:
             done
             cd ${BASEDIR}
             cp "${BASEDIR}/skupper-manifest/manifest.json" "${BASEDIR}/archives"
-            cp skupper-setup/*.yaml archives
+            cp skupper-setup/* archives
+            tar -zcf "${BASEDIR}/archives/operator-bundle-${VERSION}.tgz" operator-bundle/*
```

Comment:
In addition to the bundle directory, could you also include the bundle.Dockerfile that is produced by 'make generate-bundle'? It will be located in the top-level directory.

Reply:
Will not include bundle.Dockerfile in the tarball (as it is now) and add it as a separate artifact.

Reply:
apologies for the confusion, bundle.Dockerfile and bundle directory are included in the same tarball.